### PR TITLE
Cr 7637 add default git intg

### PR DESCRIPTION
**Pull request by roi-codefresh**

### CODE REVIEW

File: cmd/commands/common.go [hunk 1]
```diff
@@ -316,20 +317,22 @@ func getKubeContextNameFromUserSelect(cmd *cobra.Command, kubeContextName *strin
 
 func getIngressHostFromUserInput(cmd *cobra.Command, ingressHost *string) error {
 	if store.Get().Silent {
-		return nil
-	}
+		if ingressHost != nil && *ingressHost != "" {
```
Comment (roi-codefresh) [inline cmd/commands/common.go]: make field ingress required

File: cmd/commands/common.go [hunk 2]
```diff
@@ -211,6 +211,7 @@ func ensureGitToken(cmd *cobra.Command, cloneOpts *git.CloneOptions) error {
 func getGitTokenFromUserInput(cmd *cobra.Command) error {
 	gitTokenPrompt := promptui.Prompt{
 		Label: "Git provider api token",
+		Mask:  '*',
```
Comment (roi-codefresh) [inline cmd/commands/common.go:214]: use `*` instead of showing the token

**Review by oren-codefresh — APPROVED** [review]

**Pull request merged**